srv/src/main/java/my/bookshop/handlers/AdminServiceHandler.java

```diff
@@ -248,6 +256,39 @@ public void addBookToOrder(AddToOrderContext context) {
 		context.setResult(updatedOrder);
 	}
 
+	@On(event = CdsService.EVENT_UPDATE)
+	public void addBooksViaCsv(CdsUpdateEventContext context, List<Csv> csvs) {
```

Comment:
As it is a singleton we could also argue that we always expect only a single Csv in the event handler.